.travis/providers/aws-elasticbeanstalk/prepare

```diff
@@ -38,8 +38,15 @@ File.write '.ebextensions/gem_install_bundler.config', <<~str
         gem install bundler -v 2.0.1
 str
 
-# system 'bundle install --path vendor/bundle'
+File.write '.ebignore', <<~str
+  *
+  !Gemfile
+  !config.ru
+  !.ebextensions/gem_install_bundler.config
+str
+
+File.write 'secret.txt', 'secret'
 
-system 'git init'
-system 'git add .'
-system 'git commit -am "dpl test"'
+# system 'git init'
```

Comment:
Do we want to eradicate these lines altogether?

Reply:
yep, can do